File: sdkwork-ai-sdk-csharp/Models/ChatCompletionResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class ChatCompletionResponse
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public int? Created { get; set; }
        public string? Model { get; set; }
        public List<Choice>? Choices { get; set; }
        public Usage? Usage { get; set; }
        public string? SystemFingerprint { get; set; }
        public string? ServiceTier { get; set; }
    }
}

File: sdkwork-ai-sdk-go/types/chat_completion_response.go
package types

// 聊天完成响应
type ChatCompletionResponse struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Created int `json:"created"`
	Model string `json:"model"`
	Choices []Choice `json:"choices"`
	Usage Usage `json:"usage"`
	SystemFingerprint string `json:"system_fingerprint"`
	ServiceTier string `json:"service_tier"`
}
